jekyll/_cci2/deployment-integrations.adoc

```diff
@@ -0,0 +1,171 @@
+= Configuring Deploys
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+Deployment at CircleCI is just the same as any other job configuration; you create a job and configure it to deploy to virtually any target. This document provides an overview of the deployment process, along with best practices and optimization strategies.
+
+toc::[]
+
+[discrete]
+== Overview
+
+Once a software application has been developed and tested, it needs to be deployed and made available for its intended audience. CircleCI can deploy to virtualy any target, and can be easily configured to integrate with other services for QA/testing, feature management, and deployment strategies such as blue-green or canary deployment. Quickly and easily customize your config to match your requirements, whether a fully automated process or elements of manual approval are required.
+//// 
+.Deployment
+image::pipeline-to-deployment.png[Deployment]
+////
+
+== Deployment: The Basics
+
+* **Deployment Job**: To deploy your application, add a <<jobs-steps#jobs-overview,job>> to your `.circleci/config.yml` file and configure the job to run the steps you require. The <<deployment-examples#,deployment examples>> document covers example config for many common deployment targets.
+
+* **Deployment Parameters**: You will need to add environment variables/SSH keys to fulfill your deployment steps and provide authorization details and parameters.
++
+Environment variables can be added to the project itself (follow links below for detailed instructions), or defined within your configuration. An example of defining environment variables _within_ your configuration can be seen below within the deploy job config for the non-orb Heroku example:
```

Comment:
Does this come from a larger example? My instinct reading this would be to put `environment` at the root level of my config file, but I'm pretty sure it is actually supposed to be nested. Maybe this could be a link to a section with a larger example, like the ones below?

Reply:
Good point, that is confusing - fixed!